archunit/src/main/java/com/tngtech/archunit/core/domain/JavaClass.java

```diff
@@ -2355,7 +2355,27 @@ public static DescribedPredicate<JavaClass> equivalentTo(final Class<?> clazz) {
 
         /**
          * A predicate to determine if a {@link JavaClass} "belongs" to one of the passed {@link Class classes},
-         * where we define "belong" as being equivalent to the class itself or any inner/anonymous class of this class.
+         * where "belong" means that this {@link JavaClass} is equivalent to
```

Comment:
I really like the new version and think we can go with it :)

Reply:
Thanks for the additional changes after my review, @codecholeric. Looks good to me know :)